SQL/Cleanup_patches/2018-10-22-ExpireAllPasswords.sql

```diff
@@ -3,4 +3,4 @@
 UPDATE users SET Password_expiry='1990-01-01';
 -- Remove old plaintext passwords. Also affects old history entries when the
 -- column was named `hash` instead of `Password_hash`
-DELETE FROM history WHERE col LIKE "%hash%" AND tbl='users';
+DELETE FROM history WHERE col = "Password_hash" OR col = "Password_md5" AND tbl='users';
```

Comment:
This is missing brackets around the OR clause. This will be evaluated as `"Password_hash" OR col = ("Password_md5" AND tbl='users');` as is. Also, `'` is SQL standard quote for strings, not `"`.